validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -153,4 +153,24 @@ public void extra_config() {
     untilLogged(SYSTEM_CONFIG_APPLY, SYSTEM_CONFIG_APPLY_LEVEL);
   }
 
+  @Test(timeout = ONE_MINUTES_MS)
+  @Feature(stage = BETA, bucket = SYSTEM)
+  @Summary("Check that the device publishes minimum required log entries when receiving config")
+  public void config_logging() {
+    deviceConfig.system.min_loglevel = Level.DEBUG.value();
+    updateConfig("send device config set loglevel to debug");
```

Comment:
this should be a declarative statement, not an action -- so something like just "loglevel to debug", so the end output reads something like "update config loglevel to debug"

Reply:
statement updated